### PR TITLE
OHRM5X-1202: Add missing language strings in admin/pim/leave/time

### PR DESCRIPTION
**Pull request by SharukaNP**

### CODE REVIEW

**Review by SharukaNP — COMMENTED** [review]
Removed the exception translation which cause errors

File: symfony/plugins/orangehrmLeavePlugin/Controller/LeaveListController.php [hunk 1]
```diff
@@ -96,7 +98,15 @@ public function preRender(Request $request): void
      */
     protected function addLeaveStatusesProp(Component $component): void
     {
-        $component->addProp(new Prop('leave-statuses', Prop::TYPE_ARRAY, self::LEAVE_STATUSES));
+        $leaveStatuses = self::LEAVE_STATUSES;
+        $translatedLeaveStatuses = [];
+        foreach ($leaveStatuses as $leaveStatus) {
+            $translatedLeaveStatuses[] = [
+                'id' => $leaveStatus['id'],
+                'label' =>$this->getI18NHelper()->transBySource($leaveStatus['label']),
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmLeavePlugin/Controller/LeaveListController.php:106]:
```suggestion
                'label' => $this->getI18NHelper()->transBySource($leaveStatus['label']),
```
format the code

File: symfony/plugins/orangehrmPimPlugin/Controller/SaveCustomFieldController.php [hunk 2]
```diff
@@ -23,10 +23,12 @@
 use OrangeHRM\Core\Vue\Component;
 use OrangeHRM\Core\Vue\Prop;
 use OrangeHRM\Framework\Http\Request;
+use OrangeHRM\I18N\Traits\Service\I18NHelperTrait;
 use OrangeHRM\Pim\Service\CustomFieldService;
 
 class SaveCustomFieldController extends AbstractVueController
 {
+    use I18NHelperTrait;
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmPimPlugin/Controller/SaveCustomFieldController.php:31]:
```suggestion
    use I18NHelperTrait;

```
add new line

File: symfony/plugins/orangehrmPimPlugin/Controller/SaveCustomFieldController.php [hunk 3]
```diff
@@ -53,8 +55,24 @@ public function preRender(Request $request): void
...
+        $component->addProp(new Prop('screen-list', Prop::TYPE_ARRAY, $translatedScreenList));
+        $fieldTypeList =CustomFieldController::FIELD_TYPE_LIST;
+        $translatedFieldTypeList =[];
+        foreach ($fieldTypeList as $field) {
+            $translatedFieldTypeList[] =[
+                'id' => $field['id'],
+                'label' => $this->getI18NHelper()->transBySource($field['label'])
+            ];
+        }
+        $component->addProp(new Prop('field-type-list', Prop::TYPE_ARRAY, $translatedFieldTypeList));
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmPimPlugin/Controller/SaveCustomFieldController.php:75]: shall we go like this,
```php
        $component->addProp(
            new Prop(
                'screen-list',
                Prop::TYPE_ARRAY,
                array_map(
                    fn (array $screen) => [
                        'id' => $screen['id'],
                        'label' => $this->getI18NHelper()->transBySource($screen['label'])
                    ],
                    CustomFieldController::SCREEN_LIST
                )
            )
        );
        $component->addProp(
            new Prop(
                'field-type-list',
                Prop::TYPE_ARRAY,
                array_map(
                    fn (array $screen) => [
                        'id' => $screen['id'],
                        'label' => $this->getI18NHelper()->transBySource($screen['label'])
                    ],
                    CustomFieldController::FIELD_TYPE_LIST
                )
            )
        );
```

**Review by RajithaKumara — CHANGES_REQUESTED** [review]
Format all modified files using IDE

File: symfony/plugins/orangehrmPimPlugin/Controller/SaveEmployeeReportController.php [hunk 4]
```diff
@@ -521,9 +540,37 @@ public function preRender(Request $request): void
...
+                    ],
+                    $displayFields
+                )
+            )
         );
 
         $this->setComponent($component);
     }
+
+    private function translateFields(array $fields): array
```
Comment (devishke-orange) [inline symfony/plugins/orangehrmPimPlugin/Controller/SaveEmployeeReportController.php:565]: add php doc comment

**Review by Super-Chama — APPROVED** [review]

File: installer/dbscript/dbscript-2.sql [hunk 5]
```diff
@@ -4616,3 +4616,22 @@ INSERT INTO ohrm_user_role_data_group (`can_read`, `can_create`, `can_update`, `
 VALUES (1, 0, 0, 0, 0, @auth_admin_verify_data_group_id, @admin_role_id);
 
 INSERT INTO `ohrm_i18n_group` (`name`, `title`) VALUES ('attendance', 'Attendance'), ('help', 'Help'),('auth', 'Auth');
+
+ALTER TABLE `ohrm_module` ADD display_name VARCHAR(120);
```
Comment (RajithaKumara) [inline installer/dbscript/dbscript-2.sql:4620]: please note this, need to move to migration file

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**